test/smoke/generated/agrifood-data-plane/review/agrifood-data-plane.api.md

```diff
@@ -884,6 +880,10 @@ export interface CascadeDeleteJobOutput {
     status?: string;
 }
 
+// @public (undocumented)
+function createClient(Endpoint: string, options?: ClientOptions): AzureAgriFoodPlatformDataPlaneServiceClient;
```

Comment:
I think we should only export this as the default export to leave for later adding a second client, what do you think @joheredi ?

Reply:
I think this only exports it as the default below, there is no access to createClient as a named import/export 

![image](https://user-images.githubusercontent.com/21109913/159593529-dd7df6e1-06d7-4401-a77c-bde7c98f8a96.png)

Reply:
It would be nice if we could somehow make this clear in api-extractor or else api reviews are going to be misleading